admin/securedrop_admin/__init__.py

```diff
@@ -255,175 +276,176 @@ def validate(self, document):
                          "ossec@ossec.test"))
 
     class ValidateOptionalEmail(ValidateEmail):
-        def validate(self, document):
+        def validate(self, document: Document) -> bool:
             if document.text == '':
                 return True
             return super(SiteConfig.ValidateOptionalEmail, self).validate(
                 document)
 
-    def __init__(self, args):
+    def __init__(self, args: argparse.Namespace) -> None:
         self.args = args
-        self.config = {}
+        self.config = {}  # type: Dict
         # Hold runtime configuration before save, to support
         # referencing other responses during validation
-        self._config_in_progress = {}
+        self._config_in_progress = {}  # type: Dict
         translations = SiteConfig.Locales(
             self.args.app_path).get_translations()
-        translations = " ".join(translations)
+        translations_as_str = " ".join(translations)
+
         self.desc = [
```

Comment:
I changed all entries from a list to a tuple so they could be typed.

Reply:
Once Python 3.7 is supported, this would be a good candidate for using a dataclass for each entry instead of a tuple.